client/nivlheim_client

```diff
@@ -698,16 +701,26 @@ sub http_get($$) {
 	if ($httpstatus =~ m#^HTTP/1.1 [45]\d\d#) {
 		printlog("$url: $httpstatus");
 		return undef;
+	} elsif ($httpstatus =~ m#^HTTP/1.1 30[12]#) {
```

Comment:
Burde vi støtte flere statuskoder i 300-serien? Ser for meg at 307 og 308 kan bli brukt. Gjelder også http_post